express-crud/server.ts

```diff
@@ -54,7 +54,10 @@ app.post('/api/grades', async (req, res, next) => {
     if (name === undefined || course === undefined || score === undefined) {
       throw new ClientError(400, 'Name, Course, and Score are required');
     }
-    if (score < 0 || score > 100 || !Number.isInteger(score)) {
+    if (!Number.isInteger(score)) {
+      throw new ClientError(400, 'Score must be an integer');
+    }
```

Comment:
This fixes the problem, but you could also have just re-ordered the checks:
`if (score < 0 || score > 100 || !Number.isInteger(score))` would instead be:
`if (!Number.isInteger(score) || score < 0 || score > 100)`